lib/model.js

```diff
@@ -2664,7 +2664,7 @@ class Model {
    * @param  {Object}       options                         destroy options
```

Comment:
Remove all changes from this file, it is ok to have this as a documentation only warning.

Reply:
@sushantdhiman what do you think is the downside to also warning in this file?
The Sequelize documentation is large and this warning could be easily missed (IIRC I learned about `individualHooks` in the reference docs before seeing it in the tutorial docs.)  Having this warning in more places makes it more likely that devs will see it.

Reply:
API docs should not cover such warnings or expanded explanation, they should only explain working of any give option and platform limitation if any

Reply:
Okay, would it be okay for the API docs to simply say "all instances will be fetched into memory before running hooks"?  That is explaining what it does...

Reply:
`SELECT all records matching the where parameter...` it should be clear by that. SELECTED records will be loaded to memory and large SELECT surface will consume more memory

Reply:
Alright